src/plugins/irc-events/unhandled.js

```diff
@@ -14,6 +15,7 @@ module.exports = function(irc, network) {
 		network.channels[0].pushMessage(client, new Msg({
 			type: Msg.Type.UNHANDLED,
 			command: command.command,
+			commandslug: slugify(command.command),
```

Comment:
Kinda silly to do this on the server. After `webpack` PR is merged, we will be able to directly require `slugify` on the client, and use it there.

To be fair, we don't even need to use a library for this, as it can be accomplished with a simple replace.